deploy/appcluster.rc

```diff
@@ -27,6 +27,7 @@ OS_INTERFACE=public
 
 # OnBoarding Parameters
 BIGIQ_HOST=localhost
+BIGIQ_PORT=443
```

Comment:
Just wondering how about combining `BIGIQ_HOST` and `BIGIQ_PORT` since all the places referring to `BIGIQ_HOST` are suffixing `BIGIQ_PORT`. What's the benefit of declaring 2 separate value?
One single environment named `BIGIQ_ENDPOINT` or `BIGIQ_TARGET` or `BIGIQ_HOST` valued `https://localhost:443` is more simple and clear.

Reply:
It is not a priority.

Reply:
How much effort if we do that?

Reply:
Let's focus on the current scope of this commit, instead of stretching it.